### DISCUSSION TITLE
Turkish Language Leader

### DISCUSSION THREAD

**Discussion by alpdmrel** [OHF-Voice/intents#407] [Language Leaders]
Hi. I would like to apply for turkish language leader. I fluently speak turkish and english. And also can speak German.

**Comment by synesthesiam** [comment]
Thanks, @alpdmrel! We're asking new contributors to please submit at least two small PRs to become a language leader. We have empty Turkish templates in the repo ready to go :+1:

**Reply by alpdmrel** [reply]
@synesthesiam I have submitted 2 PRs.

**Comment by synesthesiam** [comment]
Thanks @alpdmrel! Adding you as language leader now :slightly_smiling_face:
